functions/New-DbaAvailabilityGroup.ps1

```diff
@@ -496,7 +492,7 @@ function New-DbaAvailabilityGroup {
                     # Add replicas
                     $null = Add-DbaAgReplica @replicaparams -EnableException -SqlInstance $second
                 } catch {
-                    Stop-Function -Message "Failure" -ErrorRecord $_ -Target $second -Continue
+                    Stop-Function -ErrorRecord $_ -Target $second -Continue
```

Comment:
Thank you Andres. I'll do my best to get to all of the AG commands this week. I'd really like to do thorough testing and our current test suite doesn't do such a good job. Does this particular change work? I always put in generic Messages because Stop-Function requires them even though it's not set as mandatory.

Reply:
I am totally unsure about the catch blocks. I just noticed that they are not consistent. I have not yet tested them. So you can just change them later or make me a suggestion, then I will change my branch and commit tomorrow.

Reply:
If you are working on New-DbaAvailabilityGroup anyway, you can also close this pull request and take from it want you want into your own branch - this is probably cleaner. Just wanted to point out a few things.